api/proto/teleport/legacy/types/events/events.proto

```diff
@@ -3228,6 +3243,8 @@ message DeviceMetadata {
   string asset_tag = 3 [(gogoproto.jsontag) = "asset_tag,omitempty"];
   // Device credential identifier.
   string credential_id = 4 [(gogoproto.jsontag) = "credential_id,omitempty"];
+  // Device origin.
+  DeviceOrigin device_origin = 5 [(gogoproto.jsontag) = "device_origin,omitempty"];
```

Comment:
fyi @codingllama, this updates `DeviceMetadata` with device origin which is required for Posthog event.